Game/Room.pde

```diff
@@ -96,7 +96,8 @@ class Room {
         if (board.board[i][j] != null) {
           pushMatrix();
           translate(i * Room.TILE, 0, j * Room.TILE);
-          shape(board.board[i][j]);
+          translate(Room.TILE/2, 0, Room.TILE/2); // to center PShape on tile
```

Comment:
Convenient for 1x1 objects but complicated for rectangular objects. 

According to the current solution, objects are formally placed in the field closest to (0,0) and occupy pxr fields. So centering the display of objects is uneffective here. See the picture and where the center of figures would be. For this solution it's better to translate the objects so they start in the point closest to (0,0). 
![image](https://user-images.githubusercontent.com/26661936/82148636-0ca4f200-9855-11ea-8254-68a30a428724.png)

Reply:
Hm, I guess you are right. I didn´t think about objects with a non 1x1 size